src/frontend/js/widgets/Dashboard/components/TeacherDashboardCourseSidebar/index.tsx

```diff
@@ -86,12 +85,11 @@ export const TeacherDashboardCourseSidebar = () => {
   );
 
   const getMenuLinkFromPath = (basePath: TeacherDashboardPaths) => {
-    const path = getRoutePath(basePath, {
-      organizationId: routeOrganizationId,
-      courseId: routeCourseId,
-      courseProductRelationId: routeCourseProductRelationId,
+    const path = getDashboardRoutePath(basePath, {
+      organizationId: routeOrganizationId ?? '',
+      courseId: routeCourseId ?? '',
+      courseProductRelationId: routeCourseProductRelationId ?? '',
```

Comment:
I tried to replace with :

```suggestion
      organizationId: routeOrganizationId,
      courseId: routeCourseId,
      courseProductRelationId: routeCourseProductRelationId,
```

and everything work's fine.

Reply:
Did you run the tests?

Reply:
i will, i had build-ts cmd to check if everything was fine.

Reply:
This does not run tests